components/tooling/glean-gradle-plugin/build.gradle

```diff
@@ -25,6 +25,11 @@ task sourcesJar(type: Jar) {
     from sourceSets.main.allSource
 }
 
+static def getLocalPublicationTimestamp() {
+    def date = new Date()
+    return date.format('yyyyMMddHHmmss')
```

Comment:
Should this function stay in sync with its equivalent in `publish.gradle`?  If so, would you mind adding a comment in each referencing the other?